.github/CHANGELOG.md

```diff
@@ -2,6 +2,43 @@
 
 <h3>New features since last release</h3>
 
+* PennyLane can now perform quantum circuit optimization using the
+  top-level transform `qml.compile`.
```

Comment:
This summary sounds a bit confusing to me, given the _other_ changelog items above introducing optimization transforms 😆 

Perhaps best to reword it in terms of what `qml.compile()` does? E.g., "combine optimization units to construct your own bespoke optimization pipeline" or something along those lines